### PR TITLE
docs for 5.0

### PR DESCRIPTION
**Pull request by zvs001**
- Moved linking to separate file.
- Added upgrading for 5.0
- updated methods on README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,147 +1,68 @@
 
 # react-native-zoom-us
 
-This is a minimum bridge of https://github.com/zoom/zoom-sdk-android and https://github.com/zoom/zoom-sdk-ios
-
-Tested on XCode 9.4.1 and node 10.14.1.
```
Comment (mieszko4) [inline README.md:6]: Could we keep information on which latest env it was tested?

File: README.md [hunk 2]
```diff
@@ -1,147 +1,68 @@
...
+This is a bridge for ZoomUS SDK:
+- android: https://github.com/zoom/zoom-sdk-android
+- ios: https://github.com/zoom/zoom-sdk-ios
 
 Pull requests are welcome.
 
+- [Example](https://github.com/mieszko4/react-native-zoom-us-test)
```
Comment (mieszko4) [inline README.md:10]: I need to update this example - it is very old

File: README.md [hunk 3]
```diff
@@ -1,147 +1,68 @@
...
 
-await ZoomUs.initialize(
-  config.zoom.appKey,
-  config.zoom.appSecret,
-  config.zoom.domain
-);
+await ZoomUs.initialize({
+  clientKey: '...',
+  clientSecret: '...',
+})
```
Comment (mieszko4) [inline README.md:36]: add `settings` showing `disableShowVideoPreviewWhenJoinMeeting` with default value

File: README.md [hunk 4]
```diff
@@ -1,147 +1,68 @@
...
 import ZoomUs from 'react-native-zoom-us';
 
-await ZoomUs.initialize(
-  config.zoom.appKey,
-  config.zoom.appSecret,
-  config.zoom.domain
-);
+await ZoomUs.initialize({
+  clientKey: '...',
+  clientSecret: '...',
```
Comment (mieszko4) [inline README.md:35]: add `domain` showing default value

File: README.md [hunk 5]
```diff
@@ -1,147 +1,68 @@
...
+  userName: 'Johny',
+  meetingNumber: '12345678',
+})
+
+// Join Meeting with extra params
+await ZoomUs.joinMeeting({
+  userName: 'Johny',
+  meetingNumber: '12345678',
+  password: '1234',
+  participantID: 'our-unique-id',
```
Comment (mieszko4) [inline README.md:58]: I've noticed an inconsistency with naming `userId` and `participantID`

**Review by mieszko4 — APPROVED** [review]
nice, thnx. finally easy procedure.

I left you few comments.

File: README.md (hunk 2 above)
Comment (zvs001) [reply README.md:10]: Ok. I just moved link from the bottom.

File: README.md (hunk 5 above)
Comment (zvs001) [reply README.md:58]: Yes. I also noticed it. It comes from SDK, so naming is ok. 
I think userId - means meeting initiator (host), participant - member

File: README.md [hunk 6]
```diff
@@ -30,11 +32,22 @@ Make sure you have appropriate description in Info.plist:
...
+  clientSecret: '...',
+})
+
+// initialize with extra config
 await ZoomUs.initialize({
   clientKey: '...',
   clientSecret: '...',
+  domain: 'zoom.us'
+}, {
+  disableShowVideoPreviewWhenJoinMeeting: true
```
Comment (mieszko4) [inline README.md:47]:
```suggestion
  disableShowVideoPreviewWhenJoinMeeting: true // ios only
```

File: README.md (hunk 5 above)
Comment (mieszko4) [reply README.md:72]: In sdk they use `userID` and `participantID` so they are consistent.
I guess we should use `userID`, too, but that would be a breaking change of version 5 (but that's just been released so I guess that would be fine)

**Review by mieszko4 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (zvs001) [reply README.md:72]: Oh, yes. You right. I thought about field name. 
Maybe in future I'll add such update with support.

**Pull request merged**